### ISSUE TITLE
Expedition file not showing in game slot

### ISSUE THREAD

**Issue by TinaMC1960** [goatfungus/NMSSaveEditor#1099]
I checked and have Java 8 and I reinstalled the Save editor but it is not showing the Expedition File in the Game Slot. It only shows the other three normal files that I have. This is the new Cursed Expedition. I know the Version of the Save Editor says Aquarius but in the past I have used it when the title was not the same as the most current state of the game. Please advise. Thank you

**Comment by mshocklee** [comment]
There are a couple previous issues open regarding a change in the NMS experimental branch that altered the save format. #1085 #1087

I expect that format change has now been incorporated in the latest update.

**Comment by TinaMC1960** [comment]
ty!

**Comment by mshocklee** [comment]
TLDR: I'm just another pleb. Check back for a save editor update.

Sorry, I could have been more specific.

The previous issues reported were found when users would attempt to use NMSSE to edit a save file from the game's experimental branch before this most recent game update.

I'm guessing that with the Cursed update, the change made by Hello Games was moved from the experimental code branch into the live game.

I'm not a developer on this project, just someone that discovered the same problem and read through the previous reported Issues. We'll just have to wait for an update to the save editor.

**Comment by TinaMC1960** [comment]
You're the best! TY for your patience : P

**Comment by goatfungus** [comment]
This should be fixed now, see version 1.17.6.

**Comment by TinaMC1960** [comment]
TY!
    On Friday, October 25, 2024 at 01:18:39 PM MDT, Brendon Matthews (GoatFungus) ***@***.***> wrote:  
 
 


This should be fixed now, see version 1.17.6.
